### PR TITLE
docs(popover): add isOpen input demo

### LINKED ISSUE #3795: docs(popover): add isOpen input demo
add demo with isOpen input(#3255)
effort1: easy(hours)

### PR DESCRIPTION
**Pull request by tarusin**
Add demo with `isOpen` input. If `isOpen = true` returns whether or not the popover is currently being shown.

Closes #3795

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+popover/demos/toggle-manual/toggle-manual.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<button type="button" class="btn btn-primary"
+        (click)="isOpen = !isOpen"
```
Comment (IlyaSurmay) [inline demo/src/app/components/+popover/demos/toggle-manual/toggle-manual.html]: what's the point? you're changing `isOpen` but you're not using it as popover's input

use `triggers=""` and `[isOpen]="isOpen"` and toggle `isOpen` by separate button

File: demo/src/app/components/+popover/demos/toggle-manual/toggle-manual.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-popover-toggle-manual',
+  templateUrl: './toggle-manual.html'
+})
+export class DemoPopoverToggleManualComponent {
```
Comment (IlyaSurmay) [inline demo/src/app/components/+popover/demos/toggle-manual/toggle-manual.ts]: change the name to `DemoPopoverTriggerByInput` (and selector, filenames and folder name as well)

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Please, add more information, what's the difference between this demo and other demos. And not clear, why "trigger input", when we call isOpen property by button click

**Comment by tarusin** [comment]
i call this demo "trigger by input" because when we click button the popover trigger by input.

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**